classes/order/Order.php

```diff
@@ -2141,7 +2141,7 @@ public function getTotalPaid($currency = null)
             }
         }
 
-        return Tools::ps_round($total, Context::getContext()->getComputingPrecision());
+        return Tools::ps_round($total, $currency->precision);
```

Comment:
hi @integritty !

Here you are replacing the computing precision with the display precision. Usually, they are the same value, but it's temporary, at some point the computing precision will always be higher than the computing precision (it's yet to be done)

if `Context::getContext()->getComputingPrecision())` doesn't return the right precision then it's probably what should be fixed...

But maybe there's something I'm not aware of? don't hesitate to explain your fix, thanx 👍

Reply:
@integritty friendly reminder